operator/src/main/java/com/oracle/coherence/k8s/operator/CoherenceOperator.java

```diff
@@ -389,6 +409,16 @@ private void assertPort(String value) {
      */
     private static final String ES_PORT = "ES_PORT";
```

Comment:
Again, we should standardize on the env var names. I don't really care which ones we use, as long as we use them consistently. Personally, I prefer shorter ones (`ES_*`), but am fine with the long ones as well.